swagger.yaml

```diff
@@ -0,0 +1,124 @@
+swagger: '2.0'
+info:
+  title: Express API Skeleton
+  description: Skeleton for Express APIs
+  version: v1
```

Comment:
Using `v1` instead of `1.0.0` so that we could remove version information from the configuration file.